### ISSUE TITLE
apiset still used in S.R.Metadata

### ISSUE THREAD

**Issue by jkotas** [dotnet/corefx#18387]
_From @danmosemsft on April 14, 2017 15:55_

@yizhang82 @stephentoub @tmat, @nguerrera @gkhanna79 is there any reason to still use apisets in this assembly? We removed them in all other places. But perhaps we need to keep it in this library because it ships OOB and does not necessarily have reverse forwarders?

```c#
// The library guards against unavailable entrypoints by using EntryPointNotFoundException.
        private static unsafe class NativeMethods
        {
            // API sets available on modern platforms:
            [DllImport(@"api-ms-win-core-file-l1-1-0.dll", EntryPoint = "ReadFile", ExactSpelling = true, SetLastError = true)]
            [return: MarshalAs(UnmanagedType.Bool)]
            internal static extern bool ReadFileModern(
                 SafeHandle fileHandle,
                 byte* buffer,
                 int byteCount,
                 out int bytesRead,
                 IntPtr overlapped
            );

            // older Windows systems:
            [DllImport(@"kernel32.dll", EntryPoint = "ReadFile", ExactSpelling = true, SetLastError = true)]
            [return: MarshalAs(UnmanagedType.Bool)]
            internal static extern bool ReadFileCompat(
                 SafeHandle fileHandle,
                 byte* buffer,
                 int byteCount,
                 out int bytesRead,
                 IntPtr overlapped
            );
        }
#pragma warning restore BCL0015
```

_Copied from original issue: dotnet/coreclr#10974_

**Comment by jkotas** [comment]
_From @danmosemsft on April 14, 2017 16:2_

Either way it's wrapped in `#pragma warning disable BCL0015 // Diasable Pinvoke analyzer errors.`. 

For some reason the Kernel32 one is not in our OneCore list (hence suppression) but is in the UWP list (WACK always allows apiset and classic versions). Need to make certain, but I believe this means we're OK for UWP though.

**Comment by jkotas** [comment]
_From @yizhang82 on April 14, 2017 16:14_

These are identical. We would only need the kernel32 version.

**Comment by jkotas** [comment]
_From @danmosemsft on April 14, 2017 16:23_

OK we should clean that up at some point.

**Comment by jkotas** [comment]
_From @gkhanna79 on April 14, 2017 16:36_

Any reason why this cannot be fixed in 2.0, as opposed to 2.1.0?

**Comment by jkotas** [comment]
_From @tmat on April 14, 2017 16:59_

Jan is making some changes: https://github.com/dotnet/corefx/pull/18356.

/cc @jkotas

**Comment by jkotas** [comment]
I have included the fix for this one in my PR.

**Comment by danmosemsft** [comment]
@jkotas if you install the Zenhub plugin it will move issues in 2 clicks.

**Comment by jkotas** [comment]
Fixed by #10974
